packages/docs/docs/components/code/index.mdx

```diff
@@ -0,0 +1,790 @@
+notice that the displayed code has a uniform color. The default highlighter uses
+[`Lezer`](https://lezer.codemirror.net/) to parse and highlighting the code but
+to do that it needs the grammar for the language you're using. You can set that
+up in your project configuration file.
+
+First, install the grammar for your language. You can check out this
+non-exhaustive [list of available grammars](/docs/grammars). For this tutorial,
+you should install the `javascript` grammar:
+
+```bash
+npm i @lezer/javascript
+```
+
+Then, in your project configuration, import the grammar and register it via
+`LezerHighlighter`:
+
+```ts title="src/project.ts"
+import {makeProject} from '@motion-canvas/core';
+import example from './scenes/example?scene';
+
+// highlight-start
+import {LezerHighlighter} from '@motion-canvas/2d';
+import {parser} from '@lezer/javascript';
+
+LezerHighlighter.registerParser(parser);
+// highlight-end
+
+export default makeProject({
+  scenes: [example],
+});
+```
+
+This sets `@lezer/javascript` up as the default parser for all your code
+snippets. If you want to use more than one language, check out the
+[Multiple Languages](#multiple-languages) section.
+
+## Defining Code
+
+The code to display is set via the [`code`](/api/2d/components/Code#code)
+property. In the simplest case, you can just use a string:
+
+```tsx editor
+import {makeScene2D, Code} from '@motion-canvas/2d';
+
+export default makeScene2D(function* (view) {
+  view.add(
+    <Code
+      fontSize={28}
+      fontFamily={'JetBrains Mono'}
+      code={'const number = 7;'}
+    />,
+  );
+});
+```
+
+However, usually code snippets contain multiple lines of code. It's much more
+convenient to use a template string for this (denoted using the backtick
+character `` ` ``):
+
+```tsx
+view.add(
+  <Code
+    fontSize={28}
+    fontFamily={'JetBrains Mono'}
+    // highlight-start
+    code={`\
+function example() {
+  const number = 7;
+}
+`}
+    // highlight-end
+  />,
+);
+```
+
+Notice two things here:
+
+- The code snippet ignores the indentation of the template string itself. The
+  template string preserves all whitespace characters, so any additional spaces
+  or tabs at the beginning of each line would be included in the snippet.
+
+- The backslash character (`\`) at the very beginning is used to escape the
+  first newline character. This lets us start the snippet on a new line without
+  actually including an empty line at the start of the snippet. Without the
+  slash, the equivalent code would have to be written as:
+  ```tsx
+  view.add(
+    <Code
+      fontSize={28}
+      fontFamily={'JetBrains Mono'}
+      // highlight-start
+      code={`function example() {
+    const number = 7;
+  }
+  `}
+      // highlight-end
+    />,
+  );
+  ```
+
+Template strings allow you to easily include variables in your code snippets
+with `${}` syntax. In the example below, `${name}` is replaced with the value of
```

Comment:
I think I prefer "with the `${}` syntax" here.

Reply:
Fixed in #1002